lua/lazyvim/config/keymaps.lua

```diff
@@ -96,6 +96,28 @@ map("n", "<leader>fn", "<cmd>enew<cr>", { desc = "New File" })
 map("n", "<leader>xl", "<cmd>lopen<cr>", { desc = "Location List" })
 map("n", "<leader>xq", "<cmd>copen<cr>", { desc = "Quickfix List" })
 
+if Util.has("trouble.nvim") then
+  local trouble = require("trouble")
+
+  map("n", "[q", function()
+    if trouble.is_open() then
+      trouble.previous({ skip_groups = true, jump = true })
+    else
+      vim.cmd.cprev()
+    end
+  end, { desc = "Previous trouble/quickfix item" })
+  map("n", "]q", function()
+    if trouble.is_open() then
+      require("trouble").next({ skip_groups = true, jump = true })
+    else
+      vim.cmd.cnext()
+    end
+  end, { desc = "Next trouble/quickfix item" })
```

Comment:
These lines should go in the trouble.nvim plugin spec as `keys`

Reply:
Done, sorry for taking so long.